src/database/migrations/20220811010242_create_donations.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema
+  .createTable('donations', (table)=> {
+      table.increments('iddonation').primary().unique();
+      table.string('name').notNullable();
+      table.string('email').unique();
```

Comment:
To vendo aqui... se for unique, a pessoa só conseguirá fazer uma doação.
Se fosse uma tabela separada (não é o caso neste projeto), dai sim seria unique telefone e email.

Reply:
Alterado, tinha esquecido esse detalhe de que não vão existir doadores no momento.